### PR TITLE
Edit website content

### PR DESCRIPTION
**Pull request by ericliu0000**
* Edited website roles content for clarity/consistency/order
* Removed one of two links to club Facebook group as they were very close to each other
* Remove some unnecessary code left in there

### CODE REVIEW

**Comment by ericliu0000** [comment]
One thing I was curious about: is it accurate to include that this club works throughout the state of NC?

File: index.html [hunk 1]
```diff
@@ -109,101 +109,32 @@ <h2>Our Roles</h2>
...
-                Facebook and Instagram to promote our products and recruit new
-                members. Additionally, they are responsible for reaching out to
+                social media to promote our products and recruit new members. 
+                Additionally, they are responsible for reaching out to
                 nonprofits to find new projects.
               </p>
             </li>
           </ul>
         </section>
-
```
Comment (ganning127) [inline index.html:138]: Would it be possible to keep this section just for the sake of in case we need the code later?

**Review by ganning127 — CHANGES_REQUESTED** [review]

**Comment by ganning127** [comment]
> One thing I was curious about: is it accurate to include that this club works throughout the state of NC?

That's what we're gonna be doing this year, instead of just local Durham nonprofits we wanna expand to all of NC!

File: index.html (hunk 1 above)
Comment (ericliu0000) [reply index.html:138]: Sure! I don't know how to revert it. But, here is a link to it: https://github.com/code4charity-ncssm/code4charity-ncssm.github.io/blob/main/index.html#L139-L206

Comment (ericliu0000) [reply index.html:138]: wait I can just add it back oops

**Comment by ericliu0000** [comment]
That sounds awesome!

File: index.html (hunk 1 above)
Comment (ericliu0000) [reply index.html:138]: it should be done now

**Comment by ganning127** [comment]
Amazing, thanks again, Eric. You should totally come to our club meetings tbh 😄

**Pull request merged**